ray-operator/apis/ray/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -6,7 +6,7 @@
 package v1alpha1
 
 import (
-	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/core/v1"
```

Comment:
```suggestion
	v1 "k8s.io/api/core/v1"
```

Linter complains about this change for some reason... https://github.com/ray-project/kuberay/actions/runs/4876658441/jobs/8702043318?pr=926#step:11:19

I think it happens when running `make test` or possibly other commands.  I should try to get to the bottom of it

Reply:
please rerun